addons/account_peppol/wizard/account_move_send.py

```diff
@@ -64,15 +80,22 @@ def _needs_ubl_cii_placeholder(self):
     # -------------------------------------------------------------------------
 
     def action_send_and_print(self, from_cron=False, allow_fallback_pdf=False):
-        # Extends the wizard to mark the move to be sent via PEPPOL first
+        # Extends 'account' to force ubl xml checkbox if sending via peppol
         self.ensure_one()
 
-        if self.checkbox_send_peppol and self.peppol_proxy_state == 'active':
-            for move in self.move_ids:
-                if not move.peppol_move_state:
-                    move.peppol_move_state = 'to_send'
+        if self.checkbox_send_peppol and not self.checkbox_ubl_cii_xml and self.enable_ubl_cii_xml:
+            self.checkbox_ubl_cii_xml = True
+
+        return super().action_send_and_print(from_cron, allow_fallback_pdf)
+
+    def _get_available_field_values_in_multi(self, move):
+        # Extends 'account' to force ubl xml checkbox if sending via peppol
+        # if there are many invoices to be sent
+        values = super()._get_available_field_values_in_multi(move)
+        if self.checkbox_send_peppol and not values.get('checkbox_ubl_cii_xml') and self._get_default_enable_ubl_cii_xml(move):
+            values['checkbox_ubl_cii_xml'] = True
```

Comment:
```suggestion
        if self.enable_peppol and self.checkbox_send_peppol:
            values['checkbox_ubl_cii_xml'] = True
```
is it not enough? :thinking: